test/Error/outofmem.js

```diff
@@ -3,18 +3,7 @@
 // Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
 //-------------------------------------------------------------------------------------------------------
 
-try
-{
-    var longString = "A";
-    for (var i = 0; i < 31; i++)
-        longString += longString;
-    WScript.Echo(longString.toString());
-}
-catch (e)
-{
-    WScript.Echo(e.name);
-    WScript.Echo(e.message);
-    WScript.Echo(e.description);
-    WScript.Echo(e.number);
-}
-
+var longString = "A";
+for (var i = 0; i < 31; i++)
+   longString += longString;
+WScript.Echo(longString.toString());
```

Comment:
My intention was indeed to add to the test and not replace it.  The original test is testing the javascript behavior of the OOM error object, ensuring that it has the `name`, `message`, `description`, and `number` properties.

Your addition to the test would then ensure that the host prints out something useful in the event of an uncaught OOM error.

Accordingly, you will need to update the baseline to match the new expected output.